tests/test_examples.py

```diff
@@ -63,6 +64,16 @@ def test_example_modules(args):
     run_main_subproc(args)
 
 
+@pytest.mark.skipif(
+    not habitat_sim.vhacd_enabled,
+    reason="Requires Habitat-sim to be built with VHACD (--vhacd)",
+)
+def test_vhacd_example():
```

Comment:
Nit can we fully paramterize with pytest.mark.parameterize so that we can add more VHACD tests easily in the future.